docs/diagrams/AddPatientSequenceDiagram.puml

```diff
@@ -0,0 +1,35 @@
+@startuml
+actor User
+User -> UI : Inputs 'add John Doe'
+activate UI
+
+UI -> Logic : Passes input to Logic
+activate Logic
+
+Logic -> Parser : parseCommand("add John Doe")
+activate Parser
+
+alt State is MAIN_STATE
+    Parser -> AddPatientCommand : new AddPatientCommand("John Doe")
+    activate AddPatientCommand
+
+    AddPatientCommand -> Model : isDuplicatePatient("John Doe")
+    activate Model
+    Model --> AddPatientCommand : Returns false (if no duplicate)
+    deactivate Model
+
+    AddPatientCommand -> Model : addPatient("John Doe")
```

Comment:
I notice that there is no activation bar for addPatient() and saveHospitalData(). It is required to be consistent, if activation bar is used, then it should be used for all